app/api/messages/api.rb

```diff
@@ -0,0 +1,10 @@
+module Messages
+  class API < Grape::API
+    prefix '/api/messages'
+    format :json
+    formatter :json, Grape::Formatter::Jbuilder
+    mount Ping
+    mount Post
+    mount Get
```

Comment:
is this the convention recommended by grape to modularize the endpoints? What do you think if we rely on REST and create one of these classes for each RESTful action, and also for each custom action (like `Ping`)?

```suggestion
    mount Ping
    mount Post
    mount Get
```
```suggestion
    mount Ping
    mount Create
    mount Show
```

Reply:
Done!